dotcom-rendering/src/types/discussion.ts

```diff
@@ -141,11 +145,60 @@ export interface CommentType {
 	};
 }
 
-export type CommentResponse = {
-	status: 'ok' | 'error';
-	statusCode: number;
-	message: string;
-	errorCode?: string;
+export type CommentResponseErrorCodes = (typeof errorCodes)[number];
+const errorCodes = [
+	'USERNAME_MISSING',
+	'EMPTY_COMMENT_BODY',
+	'COMMENT_TOO_LONG',
+	'USER_BANNED',
+	'IP_THROTTLED',
+	'DISCUSSION_CLOSED',
+	'PARENT_COMMENT_MODERATED',
+	'COMMENT_RATE_LIMIT_EXCEEDED',
+	'INVALID_PROTOCOL',
+	'AUTH_COOKIE_INVALID',
+	'READ-ONLY-MODE',
+	'API_CORS_BLOCKED',
+	'API_ERROR',
+	'EMAIL_VERIFIED',
+	'EMAIL_VERIFIED_FAIL',
+	'EMAIL_NOT_VALIDATED',
+] as const;
```

Comment:
👏

Reply:
There’s actually a lot more, but we’re not responding to them in any meaningful way